[RFC]: Reorder content

bergzand opened the pull request:
The purpose of this PR is twofold.
1. Ensure that the drop down menu matches the top navigation bar.
2. Reorder the content to ensure that it matches the content of the website.

The first issue is fixed by the first two commits.

626c5d4 removes the additional non-functional "RIOT" in the drop-down menu.
e9cfe7a Fixes the mismatch between the top navigation bar and the drop-down

The second issue can be observed in the current website by pressing the navigation buttons one by one. Observed is that the website jumps both back up and back down in a illogical manner.

17a3671 Moves the content of the website around to match the order of the navigation bar
~~f2937d1 Moves the acknowledgement to a separate section at the bottom of the page, ensuring it doesn't interrupt the information flow of the page.~~

waehlisch commented:
outdated because of relaunch.

bergzand commented:
Reopened and configured the legacy branch as target.

jia200x commented:
Since the release of the new website was discussed to be in the short term, I think it would be better to focus on bugfixes (626c5d4 and e9cfe7a) rather than enhancement/cleanups (17a3671 and f2937d1)

The fact the navbar doesn't follow the website order is in fact a bug, but I think I would be better to rearrange the navbar instead of moving the content (otherwise this falls into an enhancement). I propose then to refactor 17a3671 to rearrange the navbar and not include f2937d1.

waehlisch commented:
> The fact the navbar doesn't follow the website order is in fact a bug,

no, it is not a bug. it was intentionally designed this way to allow for my dynamic on a single-page view.

bergzand commented:
@jia200x Thanks for the review and suggestions!
- I've removed f2937d1 from this PR. 
- I've reworked 17a3671 as you suggested. The new commit reorders the navbar to match the content order.

Let me know what you think :)

emmanuelsearch commented:
I think we should close this PR on the previous version of the website. If someone disagrees, feel free to reopen!
